content/articles/creating-a-shopping-cart-api/index.md

```diff
@@ -0,0 +1,588 @@
+Building a dynamic web app, website, or mobile app requires the consumption of APIs (Application Programming Interfaces). Most companies provide APIs for carrying out some tasks such as building an eCommerce web app. For example, you can call/consume an API from the central bank of your country or some established applications like PayPal, Paystack, etc.
+<!--More -->
+In this tutorial, I will guide you through building a basic shopping cart API modeling how an actual shopping cart functions.
+In a real web app, you will be consuming APIs into this code that I have created, but for now, we have to use actual value as the amount in the shopping cart.
```

Comment:
A little unclear of what you mean here. By *actual value* do you mean hardcoded values? Also, what do you mean by *consuming APIs into this code that I have created*? Do you mean consuming other APIs within your shopping cart API? I would recommend rephrasing this.